db/seeds.rb

```diff
@@ -1,17 +1,19 @@
+require 'securerandom'
+
 puts "Seeding database"
 puts "-------------------------------"
 
 # Create an initial Admin User
 admin_username = "errbit"
 admin_email = "errbit@#{Errbit::Config.host}"
-admin_pass  = 'password'
+admin_pass  = SecureRandom.hex
```

Comment:
maybe something smaller `SecureRandom.hex[0..7]` ? 8 digts is enough i would say..